src/nvim/highlight_group.c

```diff
@@ -230,61 +230,108 @@ static const char *highlight_init_both[] = {
+  // Treesitter standard groups
+  "default link @variable            NONE",  // don't highlight to reduce visual overload
+  "default link @variable.builtin    Identifier",
+  "default link @variable.parameter  Identifier",
+  "default link @variable.member     Identifier",
+
+  "default link @constant          Constant",
+  "default link @constant.builtin  Special",
+  "default link @constant.macro    Define",
+
+  "default link @module  Module",
+  "default link @label   Label",
+
+  "default link @string                 String",
+  "default link @string.documentation   Comment",
+  "default link @string.regexp          SpecialChar",
+  "default link @string.escape          SpecialChar",
+  "default link @string.special         SpecialChar",
+  "default link @string.special.symbol  Constant",
+  "default link @string.special.url     Underlined",
+
+  "default link @character          Character",
+  "default link @character.special  SpecialChar",
+
+  "default link @boolean       Boolean",
+  "default link @number        Number",
+  "default link @number.float  Float",
+
+  "default link @type             Type",
+  "default link @type.definition  Typedef",
+  "default link @type.qualifier   StorageClass",
+
+  "default link @attribute  Function",
+  "default link @property   Identifier",
+
+  "default link @function          Function",
+  "default link @function.builtin  Special",
+  "default link @function.macro    Macro",
+
+  "default link @constructor   Special",
+  "default link @operator      Operator",
+
+  "default link @keyword            Keyword",
+  "default link @keyword.function   Statement",
+  "default link @keyword.operator   Operator",
+  "default link @keyword.import     Include",
+  "default link @keyword.storage    StorageClass",
+  "default link @keyword.repeat     Repeat",
+  "default link @keyword.debug      Debug",
+  "default link @keyword.exception  Exception",
+
+  "default link @keyword.conditional    Conditional",
+
+  "default link @keyword.directive         Preproc",
+  "default link @keyword.directive.define  Define",
+
+  "default link @punctuation.delimiter  Delimiter",
+  "default link @punctuation.bracket    Delimiter",
+  "default link @punctuation.special    Special",
+
+  "default link @comment   Comment",
+
+  "default link @comment.error    DiagnosticError",
+  "default link @comment.warning  DiagnosticWarn",
+  "default link @comment.hint     DiagnosticHint",
+  "default link @comment.info     DiagnosticInfo",
+  "default link @comment.todo     Todo",
+
+  "@markup.strong         gui=bold           cterm=bold",
+  "@markup.italic         gui=italic         cterm=italic",
+  "@markup.strikethrough  gui=strikethrough, cterm=strikethrough",
+  "@markup.underline      gui=underline,     cterm=underline",
+
+  "default link @markup.heading  Title",
+
+  "default link @markup.quote        Comment",
+  "default link @markup.math         Identifier",
+  "default link @markup.environment  Identifier",
```

Comment:
Open to better suggestions.

Reply:
I don't think _any_ standard programming language concept is a good fit for LaTeX...

I'm actually not happy about them being here at all, but here we are. I am fine just not defining them in the default scheme.

Alternative: treat them like codeblocks and use the same link as for `@markup.raw`. I think that's my preference atm.

Reply:
> Alternative: treat them like codeblocks and use the same link as for `@markup.raw`. I think that's my preference atm.

Don't have strong opinions here.